build.gradle

```diff
@@ -361,4 +361,19 @@ sonarqube {
 	}
 }
 
+// skip publishing the root module
+artifactoryPublish.skip = true
 
+artifactory {
+	contextUrl = 'https://repo.spring.io'
+	publish {
+		repository {
+			repoKey = 'libs-snapshot-local'
```

Comment:
This will move to an actual Gradle plugin (based on what spring-security) is doing in https://github.com/spring-projects-experimental/spring-pulsar/issues/20